Sources/StreamChat/WebSocketClient/EventMiddlewares/MessageReactionsMiddleware.swift

```diff
@@ -18,7 +16,7 @@ struct MessageReactionsMiddleware<ExtraData: ExtraDataTypes>: EventMiddleware {
             return
         }
         
-        database.write({ session in
+        do {
```

Comment:
There's a regression in case of `ReactionDeletedEvent` with the missing local dto: the `completion` won't be invoked